aspnetcore/fundamentals/servers/aspnet-core-module.md

```diff
@@ -1,126 +1,59 @@
 ---
 title: ASP.NET Core Module
 author: tdykstra
-description: Introduces ASP.NET Core Module (ANCM), an IIS module that lets the Kestrel web server use IIS or IIS Express as a reverse proxy server.
+description: Learn how the ASP.NET Core Module is an IIS module that allows the Kestrel web server to use IIS or IIS Express as a reverse proxy server.
```

Comment:
Grammar seems odd here. I think "Learn how the ASP.NET Core Module allows the Kestrel web server to use IIS or IIS Express as a reverse proxy server."

Reply:
There's nothing wrong with the grammar, but if the extra piece of info about the module being an IIS module isn't particularly helpful, then let's kill that bit off. I'll remove it on the next commit.